Makefile

```diff
@@ -31,8 +31,18 @@ format-check: ## Runs Format using nightly toolchain but only in check mode
 	cargo +nightly fmt --all --check
 
 
+.PHONY: taplo
+taplo: ## Runs Format for all TOML files
+	taplo fmt
+
+
+.PHONY: taplo-check
+taplo-check: ## Runs Format for all TOML files but only in check mode
+	taplo fmt --check --verbose
```

Comment:
I think similar to the workflow file, naming them `toml` instead of `taplo` would be more understandable.